Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/contestsAdminService.ts

```diff
@@ -71,6 +72,19 @@ export const contestService = createApi({
                 body: contestAdministrationModel,
             }),
         }),
+
+        getContestsExcelFile: builder.query<IFileModel, IGetAllAdminParams>({
```

Comment:
Isn't this name inconsistent with the others?

Reply:
Fixed